### PR TITLE
REGD-46-readme

### PR DESCRIPTION
**Pull request by mingjiecn**
Move local install instruction to README-local-install.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,91 +27,29 @@ pyenv local 3.8.10
...
-
-- To build **development** bundles and continue updating them as you edit source files, run:
-
-  `npm run dev`
-
-The development bundles are not minified, to speed up building. The above command runs continually in your terminal window and watches for changes in Javascript and SCSS files, rebuilding the bundles as you make changes.
-
 ## Creating a demo machine
 
+- Install denpendency boto3 first then deploy:
```
Comment (emro) [inline README.md]: This should be "dependency"

File: README.md [hunk 2]
```diff
@@ -259,20 +60,19 @@ pip3 install flake8
 npm install -g eslint
 npm install -g eslint-plugin-react
 ```
->:warning: _Note_: You don't have to use Sublime Text 3 but you *must* insure that linting in your editor behaves as it does in Sublime Text 3.
-
 
+>:warning: _Note_: You don't have to use Sublime Text 3 but you _must_ insure that linting in your editor behaves as it does in Sublime Text 3.
```
Comment (emro) [inline README.md]: This should be "ensure" not "insure"

File: README.md [hunk 3]
```diff
@@ -259,20 +60,19 @@ pip3 install flake8
...
 
 ### Sublime Text 3
```
Comment (emro) [inline README.md:66]: I'm not really sure this section on Sublime is necessary. I don't use Sublime - I use Atom - and it's just fine. Does anyone use Sublime, or VS, for that matter? Maybe we should just have a general "use linting" section.

File: README-local-install.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+## Linting your code within your code editor
+
+To set up linting with [Sublime Text 3](https://www.sublimetext.com/3) or [Visual Studio Code](https://code.visualstudio.com/), first install the linters:
+
+```bash
+pip3 install flake8
+npm install -g eslint
+npm install -g eslint-plugin-react
+```
+>:warning: _Note_: You don't have to use Sublime Text 3 but you *must* insure that linting in your editor behaves as it does in Sublime Text 3.
```
Comment (emro) [inline README-local-install.md]: Should be "ensure" not "insure"

File: README-local-install.md [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+
+
+
+### Sublime Text 3
```
Comment (emro) [inline README-local-install.md]: I commented about this in another place as well, but I'm not sure if we need these specific Sublime/VS comments. Though, they probably don't hurt.

**Review by emro — CHANGES_REQUESTED** [review]
This is a good reorganization of the Readme. I just had a couple comments: a few typos should be fixed and I'm not entirely sure that the specific sections on Sublime and VS are necessary (though they're also fine to keep, maybe they will be helpful to someone).

File: README-local-install.md [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+- The script above will spin up a server in the AWS cloud with the current branch, and with a computed nameserver alias based on the branch and your username.  There are options to use a different branch and/or different instance name and also if you want to use AWS spot instances...and you can specify which AWS profile you want to use.   
+
+  
+- Deploy script help (how to specify name, instance size, etc):
+
+  `bin/deploy --help`
+      
+For all options, including setting up ES clusters (needed for full production).  After indexing (currently 8+hrs) the machine can be downsized at AWS to an m4.2xlarge, unless you are planning to submit significant data to it.
+
+- If you don't buildout locally, you can install denpendency boto3 first then deploy:
```
Comment (emro) [inline README-local-install.md]: "dependency"

File: README.md (hunk 1 above)
Comment (mingjiecn) [reply README.md]: Will fix.

File: README-local-install.md (hunk 4 above)
Comment (mingjiecn) [reply README-local-install.md]: Will delete this part for local install. Just keep it in README

File: README-local-install.md (hunk 5 above)
Comment (mingjiecn) [reply README-local-install.md]: I personally use vs. I think we can just keep it until we have a better idea on style and formatting.

**Review by emro — APPROVED** [review]
Looks great

**Pull request merged**